src/arch/ArchHooks/ArchHooks_Win32.cpp

```diff
@@ -150,26 +150,22 @@ void ArchHooks_Win32::SetTime( tm newtime )
 
 void ArchHooks_Win32::BoostPriority()
 {
-	/* We just want a slight boost, so we don't skip needlessly if something happens
-	 * in the background.  We don't really want to be high-priority--above normal should
-	 * be enough.  However, ABOVE_NORMAL_PRIORITY_CLASS is only supported in Win2000
-	 * and later. */
-#ifndef ABOVE_NORMAL_PRIORITY_CLASS
-#define ABOVE_NORMAL_PRIORITY_CLASS 0x00008000
+	/* Enforcing HIGH_PRIORITY_CLASS is best for Win10+ where Windows Update can interrupt the game
+	 * and it also prevents streaming apps from taking rendering priority over Stepmania */
+#ifndef HIGH_PRIORITY_CLASS
+#define HIGH_PRIORITY_CLASS 0x00008000
 #endif
 
 	DWORD pri = HIGH_PRIORITY_CLASS;
-	if( IsWindowsVersionOrGreater(HIBYTE(_WIN32_WINNT_WIN2K), LOBYTE(_WIN32_WINNT_WIN2K), 0) )
-		pri = ABOVE_NORMAL_PRIORITY_CLASS;
 
-	/* Be sure to boost the app, not the thread, to make sure the
+	 /* Be sure to boost the app, not the thread, to make sure the
 	 * sound thread stays higher priority than the main thread. */
 	SetPriorityClass( GetCurrentProcess(), pri );
 }
 
 void ArchHooks_Win32::UnBoostPriority()
 {
-	SetPriorityClass( GetCurrentProcess(), NORMAL_PRIORITY_CLASS );
+	SetPriorityClass( GetCurrentProcess(), THREAD_PRIORITY_ABOVE_NORMAL );
```

Comment:
Is there a reason why `UnBoostPriority()` doesn't return to normal priority class?

Also `THREAD_PRIORITY_ABOVE_NORMAL` is not a valid argument to `SetPriorityClass()`. Did you mean `ABOVE_NORMAL_PRIORITY_CLASS`?

Reply:
> Is there a reason why `UnBoostPriority()` doesn't return to normal priority class?

To be honest, all of ITGMania/StepMania5 runs in one giant thread more or less, so I thought not letting the thread priority get too low would help with stability. you're not _supposed to_ leave a thread on high priority, but the whole game is really contained within one thread, so 🤷‍♀️ 

The other thing is this code is ancient - there's a check to see if the OS is Windows 2000 or older. I didn't think the thread priorities necessarily had to remain consistent with the way they were 20 years ago or however old this chunk of code is.
 
I'm fairly new to coding on Windows tbh so if this is not a good practice to unboost to Above Normal then I respect that. 🙂 

> Also `THREAD_PRIORITY_ABOVE_NORMAL` is not a valid argument to `SetPriorityClass()`. Did you mean `ABOVE_NORMAL_PRIORITY_CLASS`?

Yes, good catch. It compiled with the bad argument there, so I never noticed it was wrong.